markdown/templates.go

```diff
@@ -0,0 +1,85 @@
+package markdown
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+)
+
+// TemplatesDir is the directory where the markdown templates are stored
+const TemplatesDir = "./templates"
+
+// The template for shared app content
+var AppSharedTemplate *template.Template
+
+// LoadAppTemplate loads the content for a specific app
+func LoadAppTemplate(name string) (*template.Template, error) {
+	filePath := fmt.Sprintf("apps/%s", name)
+	tpl, err := loadTemplate(filePath)
+	if err != nil {
+		return nil, err
+	}
+	// add the shared template
```

Comment:
nit: We usually add a newline before comments to vertically group things a bit more.

Reply:
This should be fixed everywhere.